commands/pass.go

```diff
@@ -0,0 +1,59 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/bndw/pick/utils/pswdgen"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "pass",
+		Short: "Generate a password without storing it",
+		Long:  "The pass command is used to generate a password without storing it.",
+		Run: func(cmd *cobra.Command, args []string) {
+			runCommand(Pass, cmd, args)
+		},
+	}
+	cmd.Flags().Int("length", pswdgen.DefaultPasswordLength, "the length of the generated password")
```

Comment:
It should always use the non-interactive mode if you are about to generate more than 1 password, and the non-interactive mode otherwise.
I think this is fine and introducing a new flag just makes everything more complicated.